guide/english/csharp/array/index.md

```diff
@@ -218,6 +218,7 @@ Console.WriteLine(array.Rank); // 4
 
 * Can be easily accessed in a random manner
 * Can be easily manipulated
+* The performance will be much better because boxing and unboxing will not happen
```

Comment:
As a suggestion, a person reading this with no prior(or limited) knowledge of c# will find themselves googling the meaning of `boxing` or `unboxing`. 

I feel that some description (albeit a short one) if possible, could make this more descriptive to the viewer and will also prevent a google search from the same.
Only a suggestion, waiting to hear what the mods have to say abut this one.